canvas_modules/harness/test_resources/parameterDefs/readonly_paramDef.json

```diff
@@ -426,7 +426,19 @@
     ],
     "help": {
       "data": {
-        "url": "randomUrl"
+        "url": "randomUrl",
+        "description": {
+          "label": "Readonly Description",
+          "link": {
+            "id": "link_for_heading",
+            "data": {
+              "something": "option data"
+            },
+            "propertyId": {
+              "name": "number"
+            }
+          }
+        }
       }
```

Comment:
I think we should be using https://github.com/elyra-ai/pipeline-schemas/blob/main/common-pipeline/operators/uihints-v3-schema.json#L23.  I don't think we need to add another description object. 

We'd need a config option though that would allow users to determine if descriptions are included or not.